tests/test_arcaflow_plugin_uperf.py

```diff
@@ -100,7 +102,7 @@ def test_functional(self):
         # Test for error when port is in use
         # Make socket for port be in use.
         server = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
-        server.bind(("127.0.0.1", 20000))
+        server.bind(("127.0.0.1", 30000))
```

Comment:
Line 103 should read, "Make address _[or 'port']_ be already in use."  (The conflict is on the address, not the socket.)

Reply:
The point of the code is to ensure that the port address is already in use when the plugin tries to bind a socket to it...so, how do we phrase that nicely in 70 or so characters?  (That is, the comment needs to describe the _intention_ of the code, rather than what it actually does (which is pretty obvious).)

Reply:
I updated the comment.